tests/really-object-spec.coffee

```diff
@@ -0,0 +1,200 @@
+# Dependencies
+
+Q            = require 'q'
+ReallyObject = require '../src/really-object'
+ReallyError  = require '../src/really-error'
+protocol     = require '../src/protocol'
+
+describe 'really-object', ->
+
+  channel =
+    send: -> true
+
+  describe 'get', ->
+
+    it 'should throw error if called without passing resource parameter', ->
+      expect ->
+        object = new ReallyObject()
+        msg = object.get()
+      .toThrow new ReallyError('Can not be initialized without resource')
+
+    it 'should throw error if called with non-string resource parameter', ->
+      expect ->
+        object = new ReallyObject()
+        msg    = object.get(123)
+      .toThrow new ReallyError('Can not be initialized without resource')
+
+    it 'should resolve promise when message sent successfully', (done) ->
+      options =
+        fields: ['author']
+
+      spyOn(channel, 'send').and.callFake( () ->
```

Comment:
there's a very little tiny fix ... remove the `( )` which surround the `callFake(`. 
from all the file.